.github/workflows/release.yml

```diff
@@ -48,9 +51,16 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Build and export to Docker
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          load: true
+          tags: ${{ env.IMAGE }}:test
       
       - name: Compose up
-        run: TAG=${{ steps.meta.outputs.tags[0] }} docker compose up -d
+        run: TAG=test docker compose up -d
```

Comment:
Do we need version number appended in tag?

Reply:
no its a local tag